http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -45,7 +45,8 @@ import scala.concurrent.ExecutionContext
 import scala.concurrent.duration._
 
 case class AuthorizationError(email: Option[WorkbenchEmail] = None)
-    extends LeoException(s"${email.map(e => s"'${e.value}'").getOrElse("Your account")} is unauthorized",
+    extends LeoException(s"${email.map(e => s"'${e.value}'").getOrElse("Your account")} is unauthorized." +
```

Comment:
if `email` is `None`, does it mean a different type of error? like we can't extract user's email or something?

Reply:
I would remove the `Option` around email

Reply:
Tried to dig through to see when we would not be able to extract the user's email. 

I think the only case would be if the user's access token is invalid and doesn't return the user's email. In that case, I think we would run into an error much earlier in the process (specifically when we get to getUserInfoAndExpirationFromAccessToken). This function will spit out its own Authentication Error. So I think we can do what you recommended and remove the option since we should never get to this point